egs/pashto/v1/readme.md

```diff
@@ -0,0 +1,33 @@
+#### Pashto OCR
+
+This is a Kaldi recipe for the Pashto OCR data - a handwritten text in Arabic (or "Pashto" -  one of the two official languages of Afghanistan). At the moment limited to single isolated words.
+
+##### Database size
+The database consists of 1862 different words and two sets of authors (speakers in Kaldi-ASR):
+1. US: 12 authors, 13K samples
+2. AF: 370 authors, 420K samples
+
+
+##### Default experimental setting
+* training set: 350 AF authors (398K samples)
+* evaluation set: 20 AF authors (22K samples)
+
+##### Baseline results
+
+| Model   | %WER |                                                 |
```

Comment:
can you also create a results file including the results in more raw variant, together with the command that can be used to generate a similar output? Check other results files in egs/